### DISCUSSION TITLE
`agent-merging`: "Undo" button for agent merge

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5725] [Feature Requests]

**Comment by maxpatiiuk** [comment]
@acwhite211 this might get too complicated, but is it possible to start a transaction and let the user do a rollback for some time afterward?
And to make transactions only contain the agent merging changes - not the changes mady by other people.

Until then, our recommendation is to only give agent merging permission to people who have received some tutorial on how to use it, and back up the database before doing a lot of merging.

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/record-merging-in-specify-7/939/12
